5.x Remove error handler

markstory opened the pull request:
* Remove the deprecated parts of Error handling that have already been deprecated in 4.4
* Deprecate a big chunk of `Debugger` that is also related to error handling. It is arguable that these features should be deprecated in 4.4 and removed in 5.x and I'm open to do that if folks feel strongly about it.

markstory committed:
Remove deprecated ErrorHandler and ConsoleErrorHandler

markstory committed:
Deprecate parts of Debugger.

markstory committed:
Remove remaining references to ErrorHandler

ADmad commented:
> It is arguable that these features should be deprecated in 4.4 and removed in 5.x and I'm open to do that if folks feel strongly about it.

I would prefer doing so. There would be hardly anyone using those methods directly and shouldn't cause any grief when upgrading.

markstory commented:
> I would prefer doing so. There would be hardly anyone using those methods directly and shouldn't cause any grief when upgrading

Cool, I'll open a pull request for 4.x and update this one to be removals.

markstory commented:
4.next deprecations added in #16401

markstory committed:
Merge branch '5.x' into remove-error-handler

markstory committed:
Remove deprecated features from Debugger.

With the removal of the outputFormat options I've also removed the
ability to customize the format of `Debugger::trace()`. In the rare case
someone needs custom formatting they can use the `array` format and do
their own formatting.

markstory committed:
Remove baseline errors from phpstan.
